agent/command.go

```diff
@@ -139,7 +139,7 @@ func (a *Agent) runCommandSet(ctx context.Context, tc *taskContext, commandInfo
 				}
 			}
 		case <-ctx.Done():
-			tc.logger.Task().Errorf("Command stopped early: %s", ctx.Err())
+			tc.logger.Task().Errorf("Idle timeout duration of %f minutes has been reached, command has timed out: %s", tc.timeout.idleTimeoutDuration.Minutes(), ctx.Err())
```

Comment:
I like that you added the timeout! We may want to convert this to seconds and use an integer; floats are not always pretty.

I'm a little concerned with removing "Command stopped early" because some people may case on this in scripts or be aware of the wording. Could we keep that at the beginning of the error message (and then maybe remove "command has timed out")?